### PR TITLE
Update references of "master" in README to "main"

### PR DESCRIPTION
**Pull request by mhamdani**
Update reference of "Master in README to "main".

Details regarding this rename can be found here:
https://github.com/github/renaming

### CODE REVIEW

File: README.md
Comment (kotp) [inline README.md:137]: I know that the originals did not have the markings to make this appear as code, but making `main` look like in this comment by using the backtick can be helpful in suggesting a literal, rather than just an idea. (It is the "main" branch, but also literally the `main` branch.)

**Review by kotp — DISMISSED** [review]
Approving, but if you can make the small change, that would be appreciated as well!

File: README.md
Comment (mhamdani) [reply README.md]: No problem. Updated main to `main` in my latest commit.

**Review by kotp — APPROVED** [review]

**Comment by kotp** [comment]
You might be able to squash these commits from origin main and then do a force push up to your repository to update this PR.

**Comment by mhamdani** [comment]
> You might be able to squash these commits from origin main and then do a force push up to your repository to update this PR.

It looks like I don't have write access to this repo to merge this PR. Would you mind doing that for me?

**Comment by kotp** [comment]
That is normal... taken care of.  Thank you for the work done!

**Pull request merged**
